app/v2/controllers/request/AuthenticatedRequest.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.request
+
+import play.api.mvc.Request
+import play.api.mvc.WrappedRequest
+import v2.models.EORINumber
+
+case class AuthenticatedRequest[A](eoriNumber: EORINumber, request: Request[A]) extends WrappedRequest[A](request)
```

Comment:
Created because it includes the EORINumber as a value object, we could consider backporting EORINumber to phase 4 instead and just adjusting that auth request.